hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/ContextTransactionProcessor.java

```diff
@@ -157,6 +160,15 @@ public CallOutcome call() {
         }
     }
 
+    private void validateHevmTransaction(HederaEvmTransaction hevmTransaction) {
+        final var maxJumboEthereumCallDataSize =
+                configuration.getConfigData(JumboTransactionsConfig.class).ethereumMaxCallDataSize();
```

Comment:
Q: Will this line return something <= 6KB if `JumboEthereumTransaction` is not supported in the given Heiro network?
Or is not needed since the overall transaction size would get caught elsewhere?

Reply:
This config will return the same value regardless of whether jumbo transactions are enabled, but it shouldn't be an issue because we should be checking transaction size at ingest (this spot is contract execution handling)